pkg/cmd/root.go

```diff
@@ -47,7 +47,21 @@ var (
 var rootCmd = &cobra.Command{
 	Use:   "nitric",
 	Short: "helper CLI for nitric applications",
-	Long:  ``,
+	Long: `Nitric - The fastest way to build serverless apps
+
+To begin working with Nitric, run the 'nitric new' command:
+
+    $ nitric new
+
+This will prompt you to create a new project for your cloud and language of choice.
+
+The most common commands from there are:
+
+    - nitric run   : Run a nitric stack locally for development or testing
+    - nitric up    : Deploy code to a cloud and/or update resource changes
+    - nitric down  : Tear down your stack's resources entirely from the cloud
+
+For more information, please visit the project page: https://nitric.io/docs`,
```

Comment:
Pulumi is a great inspiration, but we should find our own voice for all of this content, lines 52, 58, 64, etc. could all be tweaked.